memlab.bib

```diff
@@ -110,6 +110,98 @@ @string{vn
 @string{vr = {Vision Research}}
 
 
+@incollection{BobrNorm75,
+	Author = {Bobrow, D. G. and Norman, D. A.},
+	Booktitle = {Representation and understanding},
+	Date-Added = {2019-02-06 15:24:22 -0500},
+	Date-Modified = {2019-02-06 15:26:04 -0500},
+	Pages = {131-149},
+	Publisher = {Elsevier},
+	Title = {Some principles of memory schemata},
+	Year = {1975}}
+
+@article{Mann19,
+	Author = {Manning, J.R.},
+	Date-Added = {2019-02-06 12:17:21 -0500},
+	Date-Modified = {2019-02-06 12:18:00 -0500},
+	Journal = {PsyArXiv},
+	Title = {Episodic memory: mental time travel or a quantummemory wave'function?},
```

Comment:
Title should be: `Episodic memory: mental time travel or a quantum 'memory wave' function?`

Reply:
Also needs a DOI (usually this goes under Volume in order for the formatting to come out correctly)